core/Config.php

```diff
@@ -703,8 +703,8 @@ class Config_LRU {
 
 	public function __construct() {
 		Deprecation::notice('4.0', 'Please use Config_MemCache instead', Deprecation::SCOPE_CLASS);
-		if (version_compare(PHP_VERSION, '5.3.7', '<')) {
-			// SplFixedArray causes seg faults before PHP 5.3.7
+		if (version_compare(PHP_VERSION, '5.3.7', '<') || version_compare(PHP_VERSION, '6.99.99', '>')) {
+			// SplFixedArray causes seg faults before PHP 5.3.7, and also in 7.0.0-RC1
```

Comment:
Yeah this needs a rebase, which isn't trivial (asset refactor work
overlaps). Will try and find some time to do it.
On Sat, 21 Nov 2015 at 5:08 AM, Daniel Hensby <notifications@github.com>
wrote:

> In core/Config.php
> <https://github.com/silverstripe/silverstripe-framework/pull/4551#discussion_r45485032>
> :
>
> > @@ -703,8 +703,8 @@ class Config_LRU {
> >
> >  	public function __construct() {
> >  		Deprecation::notice('4.0', 'Please use Config_MemCache instead', Deprecation::SCOPE_CLASS);
> > -		if (version_compare(PHP_VERSION, '5.3.7', '<')) {
> > -			// SplFixedArray causes seg faults before PHP 5.3.7
> > +		if (version_compare(PHP_VERSION, '5.3.7', '<') || version_compare(PHP_VERSION, '6.99.99', '>')) {
> > +			// SplFixedArray causes seg faults before PHP 5.3.7, and also in 7.0.0-RC1
>
> We are on RC7 now... perhaps it's sorted?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/silverstripe/silverstripe-framework/pull/4551/files#r45485032>
> .
>